views/js/controller/admin/index.js

```diff
@@ -74,9 +74,12 @@ define([
      * @param {jQuery|Element|String} container
      * @private
      */
-    var _preventManagerRemoval = function(container){
-        var $form = $(container).closest('form');
-        var $submitter = $(':submit', $form);
+    const _preventManagerRemoval = function (container) {
+        const $form = $(container).closest('form');
+        let $submitter = $(':submit', $form);
+        if (!$submitter[0]) {
+            $submitter = $('button[title=Save]', $form);
```

Comment:
If we change language settings title can also change 🤔

Reply:
Good catch, I upload a better version